org.worldpossible.ScriptLauncher.yml

```diff
@@ -0,0 +1,32 @@
+app-id: org.worldpossible.ScriptLauncher
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: worldpossible-scriptlauncher
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
```

Comment:
```suggestion
```

Apparently this is part of `fallback-x11`.

Reply:
I don't see documentation of that in the man pages, or in the source code at a glance. Are you sure?

Reply:
Nick said so under some review, I haven't checked it myself.